climada/engine/uncertainty/unc_impact.py

```diff
@@ -306,7 +306,7 @@ def plot_rp_distribution(self, figsize=(8, 6)):
         return ax
 
 
-    def plot_sensitivity_map(self, exp, salib_si='S1', figsize=(8, 6)):
+    def plot_sensitivity_map(self, exp=None, salib_si='S1', figsize=(8, 6)):
         """
         Plot a map of the largest sensitivity index in each exposure point
 
```

Comment:
I do not understand what you mean by an "Uncertainty"-map? What property of the uncertainty distribution would you like to plot on the the map?

Reply:
Also, note that all the information about the metrics uncertainty distribution is available through `self.metrics`. Thus, a user can check what you point out (which is definitely interesting to look at !)

Reply:
ah okay. I thought in addition to the sensitivity indices in each exposure point also the standard deviation of the output distribution (since you have an uncertainty distribution for each exposure point).

Reply:
A ok, that could be an idea. I think I would keep this as a possibility for the next release. I am not 100% whether it is a good idea. Plotting the standard deviation suggests that the distributions are Gaussian. However, it is a priori not clear whether this is a good assumption. For instance, in the work done by you it was shown that in general the distributions are skewed.